banks/src/main/java/org/example/BankService.java

```diff
@@ -0,0 +1,4 @@
+package org.example;
+
+public class BankService {
```

Comment:
You already declared this classes in `com.example.Bank` package (fix naming in accordance to [google code](https://google.github.io/styleguide/javaguide.html) style btw), what is the purpose of declaring exact same classes here?